tests/FSharp.Data.DesignTime.Tests/SignatureTestCases.config

```diff
@@ -33,7 +33,6 @@ Xml,TypeInference.xml,false,false,,false,
 Xml,SampleAzureServiceManagement.xml,false,false,,true,
 Xml,TimeSpans.xml,false,false,,true,
 Xml,,false,false,,false,po.xsd
-Xml,,false,false,,false,http://europa.eu/rapid/conf/RSS20.xsd
```

Comment:
Was this removed by accident?

Reply:
I removed it as `RSS20.xsd` does not exist at url anymore and was causing unit tests to fail.

I could not find an updated url.
Appologies I should have updated the PR comment.

Reply:
That URL is now retuning a failure, I think it can be removed